packages/backend/src/queryBuilder.ts

```diff
@@ -25,6 +24,9 @@ import {
 import moment from 'moment';
 import { parseAllReferences } from './exploreCompiler';
 
+const formatTimestamp = (date: Date): string =>
+    moment(date).format('YYYY-MM-DD HH:mm:ss');
```

Comment:
The timestamp format we use for the UI isn't SQL compatible so I couldn't reuse the same function.